haystack/utils/deepsetcloud.py

```diff
@@ -685,3 +803,25 @@ def get_pipeline_client(
         """
         client = DeepsetCloudClient(api_key=api_key, api_endpoint=api_endpoint)
         return PipelineClient(client=client, workspace=workspace, pipeline_config_name=pipeline_config_name)
+
+    @classmethod
+    def get_evaluation_set_client(
+        cls,
+        api_key: Optional[str] = None,
+        api_endpoint: Optional[str] = None,
+        workspace: str = "default",
+        label_index: str = "default",
```

Comment:
I'd rename that into `evaluation_set` too

Reply:
In `DeepsetCloudDocumentStore.__init__` to initalize the `EvaluationSetClient`, it would call `evaluation_set=label_index`.
Will this be to confusing?

Reply:
I think that's exactly the decision that `DeepsetCloudDocumentStore` should make. It's aware of the `label_index` and knows how to use it. The `EvaluationSetClient` on the other hand shouldn't have to deal with `label_index` at all.

Reply:
Okay I will rename it here to `evaluation_set`.